docs/requirements.txt

```diff
@@ -1,2 +1,2 @@
-sphinx==4.3.1
+sphinx==5.0.0
```

Comment:
Thank you for taking care of this upgrade! Please split it out into a separate PR though.

Reply:
Done.

See PR: 
https://github.com/AcademySoftwareFoundation/OpenCue/pull/1336